src/ai_hawk/job_manager.py

```diff
@@ -252,8 +253,10 @@ def get_jobs_from_page(self, scroll=False):
             pass
 
         try:
-            # XPath query to find the ul tag with class scaffold-layout__list-container
-            jobs_xpath_query = "//ul[contains(@class, 'scaffold-layout__list-container')]"
+            # XPath query to find the ul tag with class scaffold-layout__list
+            jobs_xpath_query = (
+                "//div[contains(@class, 'scaffold-layout__list-detail-container')]//ul"
```

Comment:
While the code works, the `scaffold-layout__list` class appears to be closer to the target ul element than `scaffold-layout__list-detail-container`, making it more appropriate for selection.